### ISSUE TITLE
Prev/Next Page Feature Request

### ISSUE THREAD

**Issue by afbora** [getkirby-v2/panel#912]
The previous and next page navigation while subpages editing would be great like file pagination on panel.

**Comment by jenstornell** [comment]
You mean go to the next / prev sibling when editing a page? Then I would like that too. Else you need to back, find the next one and then click it. Takes a few seconds on each page.

**Comment by afbora** [comment]
@jenstornell Absolutely yes, this process so boring :) Click > edit > go back > click another > edit > go back > ...

**Comment by jenstornell** [comment]
@afbora Agree!

- It takes extra time, especially for many pages.
- It takes "brain" energy by thinking what page to click on next.
- It takes motivation to keep on going, especially when clicking the wrong page.

I just started with converting a site from WP to Kirby and having just this "problem".

**Comment by lukasbestle** [comment]
I think we should build this as a field though to keep it flexible and configurable. Would be great if someone built a great one, we might then consider merging it.

**Comment by afbora** [comment]
@lukasbestle File pagination is already exist, why pages pagination is should be a extension?. I think this feature should be part of core. Not complex, just like simple file prev/next arrows.

**Comment by jenstornell** [comment]
**For completeness, file view looks like this:**

![image](https://cloud.githubusercontent.com/assets/927011/18381769/13a5dd82-767e-11e6-9891-a20e03634233.png)

**Comment by texnixe** [comment]
While this can be achieved with a field as so many other things which should not be a field, I don't think a field is the right solution for a pagination solution.

**Comment by lukasbestle** [comment]
Well, I thought about a field for two reasons:

- Fields make it possible to place the pagination wherever you want: At the top, at the bottom, somewhere in between if you have a long form
- Fields can be configured (e.g. sorting or flip rules for the order of pages)

You can't really compare this to the file pagination. Files are simply displayed alphabetically and they have a special view where there is a good spot to put the pagination arrows. Page forms aren't that simple.

**Comment by texnixe** [comment]
I agree that it is different from file pagination, not only because of the separate view but also because pagination may not even be necessary of every use case. But I still think that a blueprint setting would be better suited than yet another field, which goes out of its way to implement something that is not a form field (even if you could argue that it would be in line with fields like headline, info etc. for giving structure to your forms).

**Comment by afbora** [comment]
@lukasbestle 

 - with css, users can move the navigation wherever they want.
 - yes pages are differently configurable but Kirby has methods all about pagination. Kirby can do it 😎 

Like @jenstornell said, this feature save our days 😆

**Comment by lukasbestle** [comment]
As I wrote above, I would love to have this in the core, but the question is only *how*.

I still think that a field would make a lot of sense. As @texnixe wrote, there are already the fields `headline`, `info` and `line`. If we find a solution to display the pagination arrows at a special position instead of in a field position, a blueprint setting would of course be much more semantic and should be preferred.

**Comment by jenstornell** [comment]
**The navigation can be placed:**

1. At the sides of the form, kind of like the images.
1. In the sidebar.

>I still think that a field would make a lot of sense. As @texnixe wrote, there are already the fields headline, info and line.

I think there are major differences to `headline`, `info`, `line` ect. The main thing is that it does not belong to the form. Tabs may not seem like it belongs to tabs either but it's there to enhance the form.

In that aspect the navigation is completely outside of it. In a mind experiment we could place it below the submit button and it would still be a logical placement. That would not be possible with other form fields that belongs to the form.

I agree with @texnixe that it would fit good in the blueprint for some configuration, like sort by: modified, slug or title. Or maybe just inherit the built in sort option. Enable/disable to true might also be a good thing.

In conclusion I think that a field is an ok workaround, but only a workaround, not a real solution.

**Comment by afbora** [comment]
I think, page navigation can be placed:
 - near of search on top bar
 - right of "page settings headline"
 - left side and right side of submit button (best idea for me)

**Comment by lukasbestle** [comment]
Yes, I'd prefer such a solution too. I think we should put the pagination at the top and bottom somehow. Fixed on the left and right of the form would be great, but I don't think this is going to work well on small screens where the room is very limited on the sides.

**Comment by texnixe** [comment]
A subpages setting like:

```
pagination: true
  - sort: flip
  - position: bottom
```

**Comment by jenstornell** [comment]
In the meantime I've made a workaround for it, as a field:

https://github.com/jenstornell/kirby-panel-prevnext

![prevnext](https://cloud.githubusercontent.com/assets/927011/21147072/0e213994-c154-11e6-9fda-903039a1440e.gif)

@texnixe 

Is it possible to have both `pagination: true` and an array with options? Every other option have one or the other?

### Position

Personally I would prefer `position: top` because it's always the starting position of a page after loaded. Really easy to move from page to page. Or make it fixed like the save button, but that would require more work.

### Flip

I don't see a good case for `sort: flip`. When do we need it?

### Pagination

Maybe figure out if options are really needed here or if there is a perfect position for everyone and a perfect order, like the default sorted page order.

If there are no options, I think `pagination: true` should be default, which mean no need to set it in case we don't want it.

If we have a perfect place, do we need a disable option?

### Conclusion

Maybe we don't need anything in the blueprint at all?
